src/utils/date.ts

```diff
@@ -0,0 +1,48 @@
+import dayjs from "dayjs";
+import "dayjs/locale/pt-br";
+
+const isDateBeforeToday = (date: string | undefined) => {
+  if (!date) {
+    return false;
+  }
+  const today = dayjs().locale("pt-br");
+  const validity = dayjs(date).locale("pt-br").add(3, "hours");
+  return dayjs(today).isBefore(validity);
+};
+
+const getDateFormater = (date?: string) => {
+  const dataFormated = dayjs(date).format("DD/MM/YYYY");
+  return dataFormated;
+};
+
+const getDateDiffInDays = (date: string) => {
+  const today = dayjs().locale("pt-br");
+  const validity = dayjs(date).locale("pt-br");
+
+  const dateDiff = validity.diff(today, "day");
+  const res = dateDiff;
+  return res;
+};
+
+const getDateDiffInHours = (date: string) => {
+  const today = dayjs().locale("pt-br");
+  const validity = dayjs(date).locale("pt-br").add(3, "hours");
+
+  const HoursDiff = validity.diff(today, "hour");
+  const res = HoursDiff;
+  return res;
+};
+
+const getEndOfDayByDate = (date: Date) => {
+  return dayjs(date).endOf("day").subtract(3, "hours").toDate();
+};
+
+const date = {
+  isDateBeforeToday,
+  getDateFormater,
+  getDateDiffInDays,
+  getDateDiffInHours,
+  getEndOfDayByDate,
+};
```

Comment:
my eyes hurt 😢



exporta cada uma das funcoes e remove o export default



```suggestion
export const isDateBeforeToday = (date: string | undefined) => {
```